### PR TITLE
Add A/B test for short exhibition promo titles

### PR DESCRIPTION
**Pull request by davidpmccormick**
☝️ 

References #4418

Rather than writing several regexes to determine the outcome of `shouldRun`, I settled for having it run everywhere except at `/works/*`.

### CODE REVIEW

File: cache/edge_lambdas/toggler.js [hunk 1]
```diff
@@ -10,7 +10,15 @@
 // }
 
 // This is mutable for testing
-let tests = [];
+let tests = [
+  {
+    id: 'showShortTitles',
+    title: 'Short exhibition titles',
+    shouldRun: request => {
+      return !request.uri.match(/^\/works\/*/);
```
Comment (jamesgorrie) [inline cache/edge_lambdas/toggler.js]: Makes perfect sense, "Should run everywhere but works" - like you said.

Comment (jamesgorrie) [reply cache/edge_lambdas/toggler.js]: Although saying that, looking at the issue outlining the tests, we're only measure `/whats-on` and `/`?

Comment (jamesgorrie) [reply cache/edge_lambdas/toggler.js]: This would us avoid capturing people who don't hit other pages (from Google etc).

Comment (davidpmccormick) [reply cache/edge_lambdas/toggler.js]: Don't think I follow?

Comment (davidpmccormick) [reply cache/edge_lambdas/toggler.js]: Sticking to `/whats-on` and `/` makes sense given that's what we should be testing though.

Comment (davidpmccormick) [reply cache/edge_lambdas/toggler.js]: Would `request.uri.match(/^\/whats-on.*/) || request.uri === '/';` make sense?

Comment (jamesgorrie) [reply cache/edge_lambdas/toggler.js]: Doesn't `.*` in the what's on regex mean 1 or more characters?

**Pull request merged**